CHANGELOG_iOS.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [1.71.125](https://github.com/brave/brave-browser/releases/tag/v1.71.125)
+
+### Web3
+
+ - Added support for resolving new UD TLDs. ([#40543](https://github.com/brave/brave-browser/issues/40543))
+ - Fixed creating send transaction when sending any non-compressed Solana NFT. ([#40424](https://github.com/brave/brave-browser/issues/40424))
+
+### General
+
+ - Implemented local vector search for the selection of key text segments of page and video context. ([#36801](https://github.com/brave/brave-browser/issues/36801))
+ - Added new Playlist UI to be default. ([#40911](https://github.com/brave/brave-browser/issues/40911))
+ - Added support for running iOS app in compatibility mode on VisionOS. ([#40579](https://github.com/brave/brave-browser/issues/40579))
+ - Added “it_IT” and “es_AR” to list of supported locales for Brave News. ([#40996](https://github.com/brave/brave-browser/issues/40996))
```

Comment:
Some users might know what the above country codes mean. I would use the full names. Example:

```
 - Added Italy and Argentina support for Brave News. ([#40996](https://github.com/brave/brave-browser/issues/40996))
```

Reply:
Good call!